### ISSUE TITLE
Focus on AMP consent Banner not as expected 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#8895]
**Describe the bug**
AMP consent banner focus management does not meet a/c or screenreader UX 

**To Reproduce**
Steps to reproduce the behaviour:

1. View the banner (I used this 
2. Click ok on the privacy banner 
3. Note that the focus goes to the accept button (h2 banner heading is ignored) 
4. Click Manage my settings on the AMP consent banner 
5. Note that the focus doesn't shift to the h2 on the expanded banner 

**Expected behaviour**
[Focus management](https://paper.dropbox.com/doc/Amp-Consent-Banner-a11y-ac--BFYfVvus6LcbTIbciLWnU0hCAg-TVV52ZLGfomCNfvbmJ53f#:uid=299082894666340443569339&h2=Focus-management)
Scenario: Focus - Expanded banner 

- Given I have activated the “Manage my settings” button on the consent banner  
- When the expanded consent banner is displayed
- Then focus should be on the H2 at the top of banner e.g. (“Manage consent settings in AMP”) 

Scenario: Focus from buttons 

- Given I am on the consent banner 
- When I select to accept or reject data collection 
- Then the consent banner is closed  
- And my focus is returned to the last element that previously had focus on the page (ie. if the “ad choices / do not sell my info” link in the footer was my last focused element, then my focus should return here) 

**Screenreader UX**
Banner: https://zpl.io/VOZWqkj
Expanded banner: https://zpl.io/b6ql6ZR

**Screenshots**

**Devices and AT**

-iOS VoiceOver 
-MacOS VoiceOver 
-NVDA 
-JAWS
-Talkback on Android 

**Testing notes**

it is expected that we will need to retest this across our AT 
- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by aeroplanejane** [comment]
Branch off the base branch to merge this issue https://github.com/bbc/simorgh/tree/amp-bind-consent

**Comment by simonsinclair** [comment]
Following the merge of https://github.com/bbc/simorgh/issues/8750, this issue is now unblocked.
